src/compile.ts

```diff
@@ -88,9 +90,46 @@ export const RegisterFunctionName = `register_${FunctionlessSalt}`;
  * are added to the bound Function.
  *
  * If `<expr>` is not a Function, then the call is proxied without modification.
+ *
+ * @deprecated - ast-reflection now uses the sequence commands below.
  */
 export const BindFunctionName = `bind_${FunctionlessSalt}`;
 
+/**
+ * AST-Reflection injected s-expression commands.
+ *
+ * (stash=[COMMAND], arguments...)
+ */
+// (stash=REGISTER,stash=func,stash[]=ast,stash)
```

Comment:
I think a lone "REGISTER" string literals would be less common than an assignment

Reply:
That is what I wanted too, but the string gets stripped.